_format/1.1/index.md

```diff
@@ -37,36 +37,33 @@ Clients and servers **MUST** send all JSON:API data using this media type in the
 
 Further, the JSON:API media type **MUST** always be specified with either no
 media type parameters or with only the `profile` parameter. This applies to both
-the `Content-Type` and `Accept` headers.
+the `Content-Type` and `Accept` headers whenever they are present.
+
+The `profile` parameter is used to support [profiles].
 
 > Note: A media type parameter is an extra piece of information that can
 accompany a media type. For example, in the header
 `Content-Type: text/html; charset="utf-8"`, the media type is `text/html` and
 `charset` is a parameter.
 
-The `profile` parameter is used to support [profiles].
+> Note: These content negotiation requirements exist to allow future versions
+of this specification to add other media type parameters for extension
+negotiation and versioning.
 
 ### <a href="#content-negotiation-clients" id="content-negotiation-clients" class="headerlink"></a> Client Responsibilities
 
-Clients that include the JSON:API media type in their `Accept` header **MUST**
-specify the media type there at least once without any media type parameters.
-
 When processing a JSON:API response document, clients **MUST** ignore any
 parameters other than `profile` in the server's `Content-Type` header.
 
 ### <a href="#content-negotiation-servers" id="content-negotiation-servers" class="headerlink"></a> Server Responsibilities
 
 Servers **MUST** respond with a `415 Unsupported Media Type` status code if
-a request specifies the header `Content-Type: application/vnd.api+json`
-with any media type parameters other than `profile`.
+a request specifies the header `Content-Type: application/vnd.api+json` with
+any media type parameters other than `profile`.
 
-Servers **MUST** respond with a `406 Not Acceptable` status code if a
-request's `Accept` header contains the JSON:API media type and all instances
-of that media type are modified with media type parameters.
-
-> Note: These content negotiation requirements exist to allow future versions
-of this specification to add other media type parameters for extension
-negotiation and versioning.
+Servers **MUST** respond with a `406 Not Acceptable` status code if a request
+specifies an instance of the JSON:API media type in an `Accept` header with any
+media type parameter other than `profile`.
```

Comment:
I think there's a little nuance lost here. Perhaps something like:

> Servers **MUST** respond with a `406 Not Acceptable` status code if a
> request's `Accept` header contains the JSON:API media type and all instances
> of that media type are modified with a media type parameter other than `profile`.

Reply:
Your suggestion leaves the door open to _some_ instances of the media type using parameters other than `profile` was that your intention? If so, why?

Reply:
Ah, I see that the original allowed that too, and my suggestion is breaking that. It still seems like a problem to me though.

Reply:
Ah, I see that the original allowed that too, and my suggestion is breaking that. It still seems like a problem to me though.

Reply:
I don't think including an extra parameter as an option that some servers may understand should be rejected. Some future client (say that supports v1.2) may include a future (unknown to v1.1) parameter, and the server could choose to respond to only the v1.1 compatible parameter (`profile`).

Reply:
Thanks for the clarification. That makes sense.